app/views/newflow/login_signup/_password_show_hide.html.erb

```diff
@@ -0,0 +1,19 @@
+<span id="password-show-hide-button">
+    <span class="toggle-show-hide">SHOW</span>
+    <span class="toggle-show-hide" style="display: none">HIDE</span>
+</span>
+
+<script type="text/javascript">
+  $(window).ready(function() {
+      var pwd_input = document.getElementById('signup_password');
```

Comment:
this way, or $ way?  i see a mix of this.

Reply:
You mean this?
```
$(function() {
  // Handler for .ready() called.
});
```

Reply:
Fixed. Pushed a commit.